### PR TITLE
update zar README

### PR DESCRIPTION
**Pull request by mccanne**

### CODE REVIEW

File: cmd/zar/README.md [hunk 1]
```diff
@@ -246,14 +290,14 @@ you can see the IPs, counts, and _path strings.
 And now I can go back to my example from before and use "zar find" on the custom
 index:
 ```
-zar find -z -o - -x custom 10.164.94.120 | zq -t -
+zar find -z -x custom 10.164.94.120 | zq -t -
 ```
-Now we're talking!  And if I take the results and do a little more math to
-aggregate the aggregations, I get this:
+Now we're talking!  And if youo take the results and do a little more math to
```
Comment (philrz) [inline cmd/zar/README.md:295]:
```suggestion
Now we're talking!  And if you take the results and do a little more math to
```

File: cmd/zar/README.md [hunk 2]
```diff
@@ -71,38 +73,71 @@ different pieces and aggregate the result.
...
-output to stdout (the default if there is no output file) and pipe it to
-a plain zq command that will show the output as a table:
+This invocation of zar traverses the archive, applies the zql "count()" operator
+over each log file, and writes the output as a stream of zng data where the
+sub-streams are simply concatenated together.
+By default, the output is sent to stdout, which means you can
+simply pipe the resulting stream to
+a vanilla zq command that will show the output as a table:
 ```
 zar zq "count()" _ | zq -f table -
```
Comment (henridf) [inline cmd/zar/README.md:87]: this could just be `zar zq -f table "count()" _`

Comment (henridf) [reply cmd/zar/README.md:87]: hmm except the type headers are printed each time, which isn't ideal.

File: cmd/zar/README.md [hunk 3]
```diff
@@ -120,71 +155,82 @@ If you want to see one, just look at it with zq, e.g.
...
+## operating directly on micro-indexes
 
 Let's say instead of searching for what log chunk a value is in, we want to
-actually pull out the zng records that comprise the index.  The syntax
-is kind of clunky and we're working to clean it up but you can say...
+actually pull out the zng records that comprise the index.  This turns out
+to be really powerful in general, but to give you a taste here, you can say...
 ```
-zar find -z -o - -x zdx:type:ip 10.47.21.138 | zq -t -
+zar find -z -x zdx:type:ip 10.47.21.138 | zq -t -
```
Comment (henridf) [inline cmd/zar/README.md:213]: @mccanne this example introduces both the `-z` and the `-x` flags, but only `-z` is described in the paragraph below below. Can the example just be `zar find -z :ip=10.47.21.138 | zq -t -` ?

**Pull request merged**